internal/daemon/controller/handlers/sessions/session_service_test.go

```diff
@@ -423,14 +409,8 @@ func TestList(t *testing.T) {
 				require.Equal(len(tc.res.GetItems()), len(got.GetItems()), "Didn't get expected number of sessions: %v", got.GetItems())
 				for i, wantSess := range tc.res.GetItems() {
 					assert.True(got.GetItems()[i].GetExpirationTime().AsTime().Sub(wantSess.GetExpirationTime().AsTime()) < 10*time.Millisecond)
-					assert.Equal(1, len(wantSess.GetConnections()))
+					assert.Equal(0, len(wantSess.GetConnections())) // no connections on list
```

Comment:
We decided not to change the proto definitions for this change then?

Reply:
Probably less "decided" and more "forgot about that conversation". I can look at doing that as a refactor on top of this though. Since it might cause a bit of code churn.

Reply:
Sounds good! Personally I think I'd be in favor of a different message for the List endpoint, but we should see if there's any precedent for other list endpoints first.